From 2d5958c4133ea672b61bb01b24eadb08dcf214e6 Mon Sep 17 00:00:00 2001
From: Jon Szymaniak <jon.szymaniak@gmail.com>
Date: Thu, 16 Oct 2014 14:56:25 -0400
Subject: [PATCH 2/4] Use $(AR) in Rules.make rather than 'ar'

This is intended to ensure the ar program from the correct toolchain
is used when cross-compiling.
---
 Makefile.rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.rules b/Makefile.rules
index 22508bf..2833f65 100644
--- a/Makefile.rules
+++ b/Makefile.rules
@@ -31,7 +31,7 @@ PROGS = $(DEMO_PROGS) enhance$(SUFFIX)
 static: libtecla$(SUFFIX).a
 
 libtecla$(SUFFIX).a: $(LIB_OBJECTS)
-	ar -ru $@ $(LIB_OBJECTS); \
+	$(AR) -ru $@ $(LIB_OBJECTS); \
 	$(RANLIB) $@; \
 	rm -f $(PROGS)
 
-- 
1.9.1

